managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3929,4 +3918,13 @@ public void clearBacklogFailed(ManagedLedgerException exception, Object ctx) {
             return CompletableFuture.completedFuture(ensembles);
         });
     }
+
+    private void updateLastLedgerCreatedTime() {
```

Comment:
The method name is different from what it actually does

Reply:
This method is used to update `lastLedgerCreatedTimestamp`, updating this field indicates that the ledger has been created succesully, and at this time, we could add a scheduled task to check the ledger reaches maximum rollover time or not.

Reply:
But what it actually does is : updateLastLedgerCreatedTime And rollCurrentLedgerIfFull

Reply:
Maybe we could use the name `updateLedgerCreatedTimeAndScheduleRolloverTask`?